cypress/integration/core/ozon-content.spec.ts

```diff
@@ -369,4 +369,31 @@ describe("Ozon Content", () => {
       .invoke('removeAttr', 'interactive')
       .and('have.css', 'text-decoration-line', 'none');
   });
+
+  it('should render Figuur as dso-image-overlay', () => {
+    cy.visit('http://localhost:56106/iframe.html?id=ozon-content--figuur');
+
+    cy.get('dso-ozon-content')
+      .invoke('prop', 'content', `
+        <Figuur
+          eId="chp_13__subsec_13.7__art_13.72__table_o_1__img_o_1"
+          wId="gm1979_2__chp_13__subsec_13.7__art_13.72__table_o_1__img_o_1"
+        >
+          <Titel>Afbeelding Titel</Titel>
+          <Illustratie
+            naam="https://www.dso-toolkit.nl/master/docs/images/global.jpg"
```

Comment:
Kunnen we hier niet een relatieve plaat voor gebruiken? Zoals https://github.com/dso-toolkit/dso-toolkit/pull/1720/files#diff-9f496ee4ec21bfe10766f4ed0ea120d54800cca781dd9a24f0826d10043f53b7R65 ?

Reply:
Ja zeker een goed idee.